src/Sylius/Bundle/CoreBundle/Command/InstallDatabaseCommand.php

```diff
@@ -13,12 +13,37 @@
 
 namespace Sylius\Bundle\CoreBundle\Command;
 
+use Sylius\Bundle\CoreBundle\Command\Helper\CommandsRunner;
+use Sylius\Bundle\CoreBundle\Installer\Executor\CommandExecutor;
+use Sylius\Bundle\CoreBundle\Installer\Provider\DatabaseSetupCommandsProviderInterface;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
-final class InstallDatabaseCommand extends AbstractInstallCommand
+final class InstallDatabaseCommand extends Command
```

Comment:
As above. Can we leave the previous implementation, use for fallback and deprecate its usage?